spark/src/test/scala/org/apache/comet/exec/CometNativeReaderSuite.scala

```diff
@@ -63,4 +61,67 @@ class CometNativeReaderSuite extends CometTestBase with AdaptiveSparkPlanHelper
         |""".stripMargin,
       "select arr from tbl")
   }
+
+  /*
```

Comment:
Followup ticket to fix schema issues for `native reader - read STRUCT of ARRAY fields`